lib/config/common.ts

```diff
@@ -34,7 +34,6 @@ export interface RenovateSharedConfig {
   ignoreDeps?: string[];
   ignorePaths?: string[];
   labels?: string[];
-  managers?: string | string[];
```

Comment:
Haven't found its usages after introducing `matchManagers` in `PackageRule` interface